vendor/modules.txt

```diff
@@ -261,10 +266,9 @@ go.opencensus.io/trace/tracestate
 # go4.org v0.0.0-20201209231011-d4a079459e60
 ## explicit; go 1.13
 go4.org/bytereplacer
-# golang.org/x/net v0.5.0
+# golang.org/x/net v0.7.0
```

Comment:
Does this also fix the CVE-2022-41723? It needs golang.org/x/net update.

If yes, please add this CVE fix information to the release note.

Reply:
Done! Nice catch.